<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>480</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>选择摄像头</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cameraCBox">
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cameraBtn">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>启动</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="cameraShowLabel">
          <property name="minimumSize">
           <size>
            <width>440</width>
            <height>380</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>440</width>
            <height>380</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>识别算法：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="eigenfacesRb">
            <property name="text">
             <string>Eigenfaces</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="fisherfacesRb">
            <property name="text">
             <string>Fisherfaces</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="lbphRb">
            <property name="text">
             <string>LBPH</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="reTrainFaceBtn">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>训练识别样本</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>457</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>截图显示</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearImgBtn">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>清空显示</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="imgShowLabel">
          <property name="minimumSize">
           <size>
            <width>310</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>310</width>
            <height>300</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(200, 200, 200);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>截图保存路径（默认保存在系统图片路径）</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="savePathEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="choosePathBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="captureBtn">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>截图</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openImgBtn">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>打开</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
